### ISSUE TITLE
Add methods for model training

### ISSUE THREAD

**Issue by timesler** [timesler/facenet-pytorch#7]
The repo currently only includes models (MTCNN and InceptionResnetV1). It would be good to add code for updating pretrained models given new data or different training hyperparameters.

**Comment by shaikhaman786vector** [comment]
How to train on my own custom dataset

**Comment by timesler** [comment]
@shaikhaman786vector thank you for the prompt, I've been dragging my feet on getting this in the repo. I've just updated the repo and python package to include some utilities for finetuning the models. If you want to train on your own dataset, first update the package to the latest version on PyPI (0.2.2):

```bash
pip install --upgrade facenet-pytorch
```

After running the above command, follow [this notebook](https://github.com/timesler/facenet-pytorch/blob/training_code/examples/finetune.ipynb) to finetune a model on your own dataset. Depending on your dataset you may need to alter training parameters (e.g., learning rate, optimizer, weight decay) or data augmentation (e.g., random resizing, random horizontal flip, etc.).

If you get a chance, please let me know if you get the model training well. The finetuning notebook works well for me, but hasn't been tested much.

**Comment by shaikhaman786vector** [comment]
it will be good if you upload for triplet loss

**Comment by shaikhaman786vector** [comment]
Also if there will be code for recognizing new image, this will be complete flow for face recognition

**Comment by timesler** [comment]
@shaikhaman786vector I'm not sure what you mean by "recognizing new image". Do you mean an image for a person that was in the training set (in which case you can just pass the image tensor to the model), or an image of a new person? 

There is an example showing how to infer using a trained model in the repo (examples/infer.ipynb).
